modules/helfi_node_landing_page/helfi_node_landing_page.install

```diff
@@ -104,3 +104,17 @@ function helfi_node_landing_page_update_9001() : void {
   \Drupal::service('helfi_platform_config.config_update_helper')
     ->update('helfi_node_landing_page');
 }
+
+/**
+ * Enable hearings paragraph.
+ */
+function helfi_node_landing_page_update_9002() : void {
+  if (
+    Drupal::moduleHandler()->moduleExists('external_entities') &&
```

Comment:
No need for this condition since external entities is a dependency for `helfi_paragrahps_hearings` module.